app/src/main/java/com/saucelabs/mydemoapp/android/view/fragments/ProductCatalogFragment.java

```diff
@@ -42,6 +46,11 @@ public static ProductCatalogFragment newInstance(String param1, String param2, i
         return fragment;
     }
 
+    public ProductCatalogFragment(boolean addVisualChanges) {
+        super();
```

Comment:
I believe it's better to use this() than super() in these cases.

Reply:
There is no default constructor in that object. So this() can't be used. I'll remove it as it's probably unnecessary